M2/Macaulay2/packages/KGAlgebras.m2

```diff
@@ -0,0 +1,210 @@
+newPackage(
+    "KGAlgebras",
+    Version => "0.5",
+    Date => "February 19th, 2025",
+    Authors => {
+	{Name => "Daniel Koizumi", Email => "daniel.koizumi@utexas.edu"},
+    {Name => "Moses Samuelson-Lynn", Email => "m.samuelsonlynn@utah.edu"},
+    {Name => "Toshi Mowery", Email => "tosmow@gmail.com"}
+    },
+    Headline => "A package for KG-algebras",
+    Keywords => {"KG-algebra"},
+    DebuggingMode => false,
+    Reload=>true,
```

Comment:
The `Reload` option to `newPackage` should be to `false`, or even better, just omitted since that's the default value.